airflow/providers/amazon/aws/hooks/sagemaker.py

```diff
@@ -942,14 +942,33 @@ def _list_request(
             else:
                 next_token = response["NextToken"]
 
-    def find_processing_job_by_name(self, processing_job_name: str) -> bool:
-        """Query processing job by name"""
```

Comment:
Removing this function is a breaking change.
Can we deprecate it first with backward compatibility?
if not we need to set entry in the change log for this one

Reply:
I'll add it back in with a deprecation notice.